src/modules/footer/components/attributionInfo/AttributionInfo.js

```diff
@@ -0,0 +1,97 @@
+import { html, nothing } from 'lit-html'; 
+import { BaElement } from '../../../BaElement';
+import { $injector } from '../../../../injection';
+import css from './attributionInfo.css';
+
+/**
+ * a class for displaying the attribution of the basemap
+ * @class
+ * @author bakir_en 
+ */
+export class AttributionInfo extends BaElement {
+
+	constructor() {
+		super();
+		const { TranslationService, GeoResourceService } = $injector.inject('TranslationService', 'GeoResourceService');
+		this._translationService = TranslationService;
+		this._georesourceService = GeoResourceService;
+	} 
+
+
+	/**
+     * @override 
+     */
+	createView() {
+		const translate = (key) => this._translationService.translate(key);
+		const { active, zoom } = this._state;
+
+		if (typeof active !== undefined && active.length > 0) {
```

Comment:
Unneeded check: `active` is guaranteed not to be null or undefined by the layers reducer